src/topology/algebra/ordered.lean

```diff
@@ -1197,6 +1197,26 @@ pi_Ioo_mem_nhds ha hb
 
 end pi
 
+lemma disjoint_nhds_nhds [densely_ordered α] {c d : α} (hcd : c < d) :
+  disjoint (𝓝 c) (𝓝 d) :=
```

Comment:
I had originally used these in some of the proofs, but they got taken out over the process of golfing the proofs. Since it doesn't sound like these lemmas are very generally useful I've just removed them.